routing/cross_routing_context.cpp

```diff
@@ -99,18 +110,24 @@ void CrossRoutingContextReader::Load(Reader const & r)
 bool CrossRoutingContextReader::ForEachIngoingNodeNearPoint(ms::LatLon const & point, function<void(IngoingCrossNode const & node)> && fn) const
 {
   bool found = false;
-  m_ingoingIndex.ForEachInRect(m2::RectD(point.lat - kMwmCrossingNodeEqualityRadiusDegrees,
-                                     point.lon - kMwmCrossingNodeEqualityRadiusDegrees,
-                                     point.lat + kMwmCrossingNodeEqualityRadiusDegrees,
-                                     point.lon + kMwmCrossingNodeEqualityRadiusDegrees),
-                               [&found, &fn](IngoingCrossNode const & node)
-                               {
+  m_ingoingIndex.ForEachInRect(GetMwmCrossingNodeEqualityRect(point), [&found, &fn](IngoingCrossNode const & node) {
                                  fn(node);
                                  found = true;
                                });
   return found;
 }
 
+bool CrossRoutingContextReader::ForEachOutgoingNodeNearPoint(ms::LatLon const & point,
```

Comment:
Yes, we can. I rewrote as a templates two functions: ForEachIngoingNodeNearPoint and ForEachOutgoingNodeNearPoint